cmd/run.go

```diff
@@ -414,6 +388,36 @@ a commandline interface for interacting with it.`,
 	},
 }
 
+func reportUsage(execScheduler *local.ExecutionScheduler) error {
+	execPlan := execScheduler.GetExecutionPlan()
+	executorConfigs := execScheduler.GetExecutorConfigs()
+	execState := execScheduler.GetState()
+
+	executors := make(map[string]int)
+	for _, ec := range executorConfigs {
+		executors[ec.GetType()]++
+	}
+
+	body, err := json.Marshal(map[string]interface{}{
+		"k6_version": consts.Version,
+		"executors":  executors,
+		"vus_max":    lib.GetMaxPossibleVUs(execPlan),
```

Comment:
I think we also would want to see what the actual initialized VU count is and how much it differs from the preallocated VU count.

Reply:
I added the initialized VU count in f271e476, but I'm wondering if we should add more data to this. The preallocated VU count is ony relevant for the arrival rate executors, no? And isn't this stateful information that will change throughout the test? What I mean is, why does it matter how many initialized/preallocated VUs there were at the end of the test? That value is not that useful by itself, unlike `MaxVUs`, which remains constant.

I find even the executor count as too much detail, though that at least would give an idea which executors are most widely used. Anything more starts to resemble analytics-level of reporting, which could be too intrusive for comfort.

Anyway, if you guys feel strongly about it, please post an example payload or schema of what you have in mind.

Reply:
>  Anything more starts to resemble analytics-level of reporting, which could be too intrusive for comfort.

Fully agree, so as I said, as little information as possible. I don't even really think we should have `lib.GetMaxPossibleVUs(execPlan),`, rather `vus_max` can just be `execState.GetInitializedVUsCount()`. It's much more useful to know how many VUs were actually used...